lasagne/nonlinearities.py

```diff
@@ -22,10 +34,30 @@ def rectify(x):
 
 
 # leaky rectify
-# Maas et al: Rectifier Nonlinearities Improve Neural Network Acoustic Models
-# http://web.stanford.edu/~awni/papers/relu_hybrid_icml2013_final.pdf
 class LeakyRectify(object):
+    """Implementation of a leaky rectifier.
+
+    Parameters
+    ----------
+    leakiness : float in [0, 1]
+        A leakiness of 0 will lead to the standard rectifier,
+        a leakiness of 1 will lead to a linear activation function.
+
+    Methods
+    -------
+    __call__(x)
+        Calculate the neurons output by applying this leaky rectifier to the
+        activation `x`.
 
+    References
+    ----------
+    The leaky rectifier is described in [1]_.
+
+    .. [1] Maas et al, "Rectifier Nonlinearities Improve Neural Network
+       Acoustic Models",
+       http://web.stanford.edu/~awni/papers/relu_hybrid_icml2013_final.pdf
```

Comment:
Right... maybe someone else can figure out what's going on here. In the meantime I don't think this is a showstopper so let's ignore it for now.